jkube-kit/generator/api/src/test/java/org/eclipse/jkube/generator/api/support/AbstractPortsExtractorTest.java

```diff
@@ -19,21 +19,26 @@
 import org.eclipse.jkube.kit.common.JavaProject;
 import org.eclipse.jkube.kit.common.PrefixedLogger;
 import org.eclipse.jkube.kit.common.util.FileUtil;
-import mockit.Expectations;
-import mockit.Mocked;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.entry;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
-
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Don't use `@RunWith` annotation. Instead mock these objects in `@Before` `setUp` method